_haikus/hacktoberfest

```diff
@@ -0,0 +1,9 @@
+---
+layout: hailus
+title: Hacktoberfest
+Author: myself
+---
+
+*Its Hacktoberfest
+Time to make more pull reqests
```

Comment:
Make sure to add a single space or <br> at the end of each line for line breaks.